src/ExpressionEvaluator/Core/Source/Concord/Concord.csproj

```diff
@@ -39,6 +39,7 @@
           Inputs="@(ConcordAssemblies->'$(ConcordDir)%(Identity).il')"
           Outputs="$(OutDir)%(Identity).dll">
     <Exec Command="$(MSBuildFrameworkToolsPath)ilasm.exe /dll /quiet /mdv:v2.0.50727 &quot;/output:$(OutDir)@(ConcordAssemblies).dll&quot; $(ConcordDir)@(ConcordAssemblies).il" />
+    <Exec Command="$(VSLOutDir)\FakeSign.exe &quot;$(OutDir)@(ConcordAssemblies).dll&quot;" />
```

Comment:
I believe we need to change the Concord project to depend on FakeSign.csproj.  That way it's guaranteed to be in the output directory.  Without adding this a direct build of Concord will fail.